README.md

```diff
@@ -14,14 +14,14 @@ A text file in JSON format representing input data, e.g.
 
 ```jason
 {
-  "rowNumber": 3,
-  "colNumber": 3,
-  "rows": [
+  "sizeRows": 3,
```

Comment:
Better than rowNumber, but from my point of view the most intuitive is: numOfRows, numOfCols.

Reply:
One of the reasons for this refactor is to get rid of 'number' in names of this parameters. Every value there is a number...

Reply:
I agree with @jzych, in my opinion, size is a better version and more intuitive than the number.

Reply:
Ok, thanks for opinion @OriaTori. Last question: what about changing:
sizeRows -> rowsSize? 
you are using below functions with names: getRowsSize()

Reply:
I think it would be better to change getter functions names to getSizeRows(), as you mention earlier that their names are not corresponding exactly with variable name.

Reply:
Renamed to getSizeCols/Rows.